calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php

```diff
@@ -238,13 +237,20 @@ protected function compile()
 		$objTemplate->hasDetails = false;
 		$objTemplate->hasTeaser = false;
 
-		// Tag the response
+		// Tag the event and its child content elements (see #2137)
 		if (System::getContainer()->has('fos_http_cache.http.symfony_response_tagger'))
 		{
-			/** @var ResponseTagger $responseTagger */
-			$responseTagger = System::getContainer()->get('fos_http_cache.http.symfony_response_tagger');
-			$responseTagger->addTags(array('contao.db.tl_calendar_events.' . $objEvent->id));
-			$responseTagger->addTags(array('contao.db.tl_calendar.' . $objEvent->pid));
+			$arrTags = array('contao.db.tl_calendar_events.' . $objEvent->id);
+
+			if ($objElements = ContentModel::findPublishedByPidAndTable($objEvent->id, 'tl_calendar_events'))
```

Comment:
Same here, I don't think you need to tag the content elements.

Reply:
Adjusted in 4b61ac2659d1dbb4d2809edd52acb6c9d468c4b0.